pkg/image/build.go

```diff
@@ -44,6 +49,11 @@ func Build(cfg *config.Config, dir, imageName string, secrets []string, noCache,
 	_ = os.Remove(bundledSchemaFile)
 	_ = os.Remove(bundledSchemaPy)
 
+	err := checkCompatibleDockerIgnore(dir)
+	if err != nil {
```

Comment:
looks like this and below could use an `iferr` fix to prevent variable redeclaration

Reply:
Resolved in [e474b84](https://github.com/replicate/cog/pull/2210/commits/e474b8492886f6521760f8277c1f1c44f4eb916f)